features/306.ngsild.BigDataSpark.feature

```diff
@@ -0,0 +1,132 @@
+
+  # After sending data for Tractor and Devices, we wait one minute until we could have some
+  # output.
+  #
+  Scenario: Logger - checking the output
+    Given I wait "60" seconds
+    When  I Compare next lines in terminal spark-run-jar at least I can find 1 in stdout with a timeout 10 matching filename 01.expected_on_terminal.txt
+    And   I flush the terminal spark-run-jar queues
+
+  # - Logger - Installing the JAR - feedback-ld - 2nd example
+  Scenario: Run Feedback Loop - Persisting Context Data
+    Given I prepare the script run-spark-feedback-ld.sh to be run
+    When  I open a new shell terminal run-spark-feedback-ld and run "features/data/306.ngsild.Big_Data_Spark/run-spark-feedback-ld.sh"
+    And   I wait "2" seconds
+    Then  everything is ok
+
+  # Request 3 -- register
+  Scenario Outline: Registering spark logger
+    When I prepare a POST HTTP request for "<description>" to "http://localhost:1026/ngsi-ld/v1/subscriptions/"
+    And  I set header Content-Type to application/ld+json
+    And  I set header NGSILD-Tenant to openiot
+    And  I set the body request as described in <file>
+    And  I perform the request
+    Then  I receive a HTTP "201" response code
+    Examples:
+      | description                | file            |
+      | notify of animal locations | 03.request.json |
+
+   # Request 4 -
+  Scenario: Check the new subscritpion for Spark logger
+    When  I send GET HTTP request to "http://localhost:1026/ngsi-ld/v1/subscriptions/"
+    And   I set header NGSILD-Tenant to openiot
+    Then  I receive a HTTP "200" response code
+
+  # Populate humidity sensors....
+  # In order to get some data relative to humidity
+  # This is not in tutorial, but it is the way to add data without Web dashboard
+  ## Example post: 12:53:47 PM HTTP POST http://iot-agent:7896/iot/d?i=filling001&k=854782081 f|0.45
+  Scenario Outline: Communicating with IoT Devices: Using Actuators
+    When  I prepare a POST HTTP request to "http://localhost:7896/iot/d?k=<key_value>&i=<sensor>"
+    And   I set header fiware-service to openiot
+    And   I set header fiware-servicepath to /
+    And   I set header Content-Type to text/plain
+    And   I set simple sensor values as described in "<sensor_value>"
+    And   I perform the request
+    Then  I receive a HTTP "200" response code
+    And   I wait "1" seconds
+    Examples:
+      | sensor_value | key_value   | sensor      |
+      | h\|73        | 548027571   | humidity001 |
+      | h\|78        | 548027571   | humidity001 |
+      | h\|83        | 548027571   | humidity001 |
+      | h\|88        | 548027571   | humidity001 |
+
+  # After sending data for Tractor and Devices, we wait one minute until we could have some
+  # output.
+  Scenario: Logger - checking the output
+    Given I wait "60" seconds
+    When  I Compare next lines in terminal run-spark-feedback-ld at least I can find 1 in stdout with a timeout 10 matching filename 02.expected_on_terminal.txt
```

Comment:
The same comment that was described before

Reply:
Again this label is that way from previous branches and it is in the develop branch. I suggest setting a refactor branch for this changes in the labels which are already integrated in the develop branch.

Reply:
See the redefinition of the scenario as it was described in PR #32

Reply:
A technical debt ()issue #34) has been defined to tackle with this issue.

NTC